pkg/image/scan.go

```diff
@@ -307,7 +308,29 @@ func (h *ScanHandler) getOrCreateGallery(ctx context.Context, f file.File) (*mod
 		return h.getOrCreateZipBasedGallery(ctx, f.Base().ZipFile)
 	}
 
-	if h.ScanConfig.GetCreateGalleriesFromFolders() {
+	// Read files in Folder to find out if the Folder is marked to be handled differently as the setting
+	folderPath := filepath.Dir(f.Base().Path)
+	folder, err := os.Open(folderPath)
+	if err != nil {
+		return nil, fmt.Errorf("Could not open Folder %s: %w", folderPath, err)
+	}
+	files, err := folder.Readdir(0)
+	if err != nil {
+		return nil, fmt.Errorf("Could not read Files in Folder %s: %w", folderPath, err)
+	}
+
+	forceGallery := false
+	exemptGallery := false
+	for _, file := range files {
```

Comment:
Rather than loop through possibly thousands of files in a folder, just check if the files you are looking for exist.
https://stackoverflow.com/questions/12518876/how-to-check-if-a-file-exists-in-go

Reply:
I was hoping for a case insensitive structure, to minimize usage problems. However I see your point. 
I've compromised by checking for a list of filenames, including camelcase, capitalized and lowercase writing. Those should probably take into consideration all scenarios while still reducing the actual looping to a low static value.

Is that acceptable?

Reply:
LGTM

Reply:
I think there's little value in checking various combinations of case. My preference would be to make it all lowercase and clearly document it.

Reply:
Well, can't say I'm agree obviously, but I won't argue over it, its really not that important. Changed accordingly documentation and code.